tests/test_quality.py

```diff
@@ -96,3 +96,35 @@ def test_resample_nu_quality_sweep(sr_orig, sr_new, fil, rms):
 
     err = np.mean(np.abs(y[:-1][idx] - y_pred[idx]))
     assert err <= rms, '{:g} > {:g}'.format(err, rms)
+
+
+@pytest.mark.parametrize('sr_orig,sr_new', [(44100, 22050), (22050, 44100)])
+def test_quality_sine_parallel(sr_orig, sr_new):
+    FREQ = 512.0
+    DURATION = 2.0
+    FILTER = 'sinc_window'
+    RMS = 1e-13
+
+    x, _ = make_tone(FREQ, sr_orig, DURATION)
+    y_seq = resampy.resample(x, sr_orig, sr_new, filter=FILTER, parallel=False)
+    y_par = resampy.resample(x, sr_orig, sr_new, filter=FILTER, parallel=True)
+
+    err = np.sqrt(np.mean(np.abs(y_seq - y_par)**2))
+    assert err <= RMS, '{:g} > {:g}'.format(err, RMS)
```

Comment:
These tests are probably better using `np.allclose` with a suitable tolerance.  That way it will check sample-by-sample rather than in aggregate.

Reply:
yep, should be fixed now